### PR TITLE
Doc: Fix typo

### PR DESCRIPTION
**Pull request by kubinpetr**
Use same function name as in class example below.

### CODE REVIEW

File: docs/en/index.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ $articles = $entityManager->getDao(App\Article::getClassName());
 
 $article = new Article();
 $article->title = "The Tigger Movie";
-$articles->save($article);
+$articles->publish($article);
```
Comment (fprochazka) [inline docs/en/index.md]: this is method of the repository, not the service

Comment (kubinpetr) [inline docs/en/index.md]: Ohh shit.....

Comment (fprochazka) [inline docs/en/index.md]: If you wanna help, change the usage to EntityManager->flush(), I don't wanna advertise the save method anyway :)

**Comment by fprochazka** [comment]
I've rewritten it a bit. Thank you anyway :)

**Comment by kubinpetr** [comment]
Thanks for nothing. Sorry, but I am not good to help you with this. Doctine is still hard for me, but I am trying to improve. :)

**Comment by fprochazka** [comment]
You've helped motivating me to actually do it, so that is a plus :)

**Pull request closed without merging**